### DISCUSSION TITLE
About HW6

### DISCUSSION THREAD

**Discussion by Maid-Unknown** [cooperrc/advanced-dynamics#45] [Q&A]
I really don't understand  what kind of form is required or Question 3 and 4. What is the feedback should be looks like? Is like [0,1,2,3} or just one formular?

**Comment by mazzzerati** [comment]
I believe you'll want to have df = an array of the derivative with respect to q1, q2, and q3. Then for df/dt you'll want to create the three terms (df/dq, dq/dt, and df/dt) that go into df/dt. You can see the formula for df/dt below the cell: df/dt = (df/dq)*(dq/dt)+df/dt.
